src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -98,7 +98,7 @@
 		<loan-use
 			v-if="!isLoading"
 			class="tw-mb-2.5"
-			loan-use-max-length="52"
+			:loan-use-max-length="52"
```

Comment:
This was going through as a string causing console errors of a mis-matched prop type.